src/jumpstart/JumpstartIntro.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView, useSafeAreaInsets } from 'react-native-safe-area-context'
+import BackButton from 'src/components/BackButton'
+import CustomHeader, { CUSTOM_HEADER_HEIGHT } from 'src/components/header/CustomHeader'
+import Leaf from 'src/images/Leaf'
+import WaveCurve from 'src/images/WaveCurve'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  children?: React.ReactNode
+  button: React.ReactNode
+  showNoAssetsHint?: boolean
+}
+
+export default function JumpstartIntro({ button, children, showNoAssetsHint }: Props) {
+  const { t } = useTranslation()
+  const insets = useSafeAreaInsets()
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer} edges={['top']}>
+      <Leaf style={styles.palmImage} />
+      <WaveCurve style={styles.waveImage} />
+      <CustomHeader style={styles.header} left={<BackButton />} />
+
+      <ScrollView contentContainerStyle={[styles.container, { paddingBottom: insets.bottom }]}>
+        <Text style={styles.title}>{t('jumpstartIntro.title')}</Text>
+
+        <View style={styles.description}>
+          <Text style={styles.descriptionLine}>{t('jumpstartIntro.description')}</Text>
+          {showNoAssetsHint && (
+            <Text style={styles.descriptionLine}>{t('jumpstartIntro.noFundsHint')}</Text>
+          )}
+        </View>
+
+        {button}
+      </ScrollView>
+
+      {children}
+    </SafeAreaView>
+  )
+}
+
+const styles = StyleSheet.create({
+  safeAreaContainer: {
+    flex: 1,
+  },
+  header: {
+    paddingHorizontal: Spacing.Thick24,
+  },
+  container: {
+    paddingHorizontal: Spacing.Thick24,
+    flex: 1,
+    gap: Spacing.Regular16,
+    justifyContent: 'center',
+    marginTop: -CUSTOM_HEADER_HEIGHT,
```

Comment:
do we need this? this would offset the whole container to the top, but we already have some inset padding at the bottom, creating too much space on the bottom? not that it really matters here for the type of content we have. i think i tend to question negative margins (they sometimes behave unpredictably on android) and f it's not too consequential here, maybe we can remove

Reply:
@kathaypacific I've noticed that in figma the text is perfectly in the centre:
- 210px from the top of the screen to the title
- 210px from the bottom of the button to the bottom of screen

But when the `CustomHeader` is included - it shifts the container and the center now is ~50px lower. That's the reason why I've set the margin to negative value. 
Also, bottom insets on iPhone SE 3rd gen is 0 so it won't be compensated.

Reply:
yeah the bottom insets are only non zero if there is no hardware buttons on the bottom of the screen (e.g. iphone 15 pro). i'm not sure if the paddingBottom of the insets would throw off the centering of the content, but it's not a big issue

Reply:
perhaps another thing you could do to avoid this negative margin is to not use the custom header, and instead use an absolutely positioned back button component